integrationTests/testProcessorNode.go

```diff
@@ -1282,7 +1282,7 @@ func (tpn *TestProcessorNode) initMetaInnerProcessors() {
 				MinStepValue:                         "10",
 				MinStakeValue:                        "1",
 				UnBondPeriod:                         1,
-				StakingV2Epoch:                       StakingV2Epoch,
+				StakingV2Epoch:                       0,
```

Comment:
Add comment to the StakingV2Epoch const.

Reply:
done